Automattic/wc-calypso-bridge
# Add avalara plugin to Tax task

### Changes proposed in this Pull Request:



Closes #996.

Add AvaTax from tax task in core to bridge

- If AvaTax is Installed but not activated - activate and then navigate user to AvaTax settings
- If AvaTax is Installed and activated - navigate user to AvaTax settings
- We don't handle where the plugin is not installed.






### How to test the changes in this Pull Request:




1. Choose an Avalara AvaTax and  WooCommerce Tax supported country such as US
2. Go to `Go to WooCommerce > Home > Tax task`
3. Observe that `Good news! Jetpack and WooCommerce Tax can automate your sales tax calculations for you.` screen is shown.
4. Download `Avalara AvaTax` plugin from https://woocommerce.com/my-account/downloads/
5. Install `Avalara AvaTax` plugin (Upload the zip file to your free trial site and run `wp plugin install woocommerce-avatax.zip`)
6. `Go to WooCommerce > Home > Tax task`
7. Confirm that Avalara AvaTax appears as one of the available tax plugins.
8. Click on Avalara > `Enable & setup` button
9. Confirm the Avalara plugin is activated.
10. Observe that you're redirected to `/wp-admin/admin.php?page=wc-settings&tab=tax&section=avatax`
11. `Go to WooCommerce > Home > Tax task` again
12. Click on Avalara > `Continue setup`  button
13. Observe that you're redirected to `/wp-admin/admin.php?page=wc-settings&tab=tax&section=avatax`

If you're getting the `There was a problem activating the plugin. Please try again.` error, please make your wpcomsh contains https://github.com/Automattic/wpcomsh/pull/1301 changes.

![Screenshot 2023-03-09 at 12 24 18](https://user-images.githubusercontent.com/4344253/223916486-87ff10e2-4505-4cc1-9c6b-bdb95ae20704.png)

![Screenshot 2023-03-07 at 17 40 05](https://user-images.githubusercontent.com/4344253/223383490-6939d3ed-7296-457e-804d-e8f940b4efc5.png)

![Screenshot 2023-03-09 at 12 25 08](https://user-images.githubusercontent.com/4344253/223916540-1c6fe31f-1ca8-48b6-8349-6722082b2b04.png)




### Other information:




### FOR PR REVIEWER ONLY:
